### PR TITLE
Adding Urbana file

### LINKED ISSUE #29: Create a new file for brianamarie
## Add file for Urbana, IL 🌽
- [x] Create a branch 🌴 
- [x] Add my file 📎 
- [x] Create a commit 💬 
- [x] Open a pull request 📍 
- [ ] Have a discussion about the pull request 😮 
- [ ] Merge our pull request 👍

### PR DESCRIPTION
**Pull request by brianamarie**
In case anyone wants to know what to do and eat in Urbana, IL 😄 

This pull request closes #29 ✨ 🌈 💓

### CODE REVIEW

File: places/urbana.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Urbana, IL
+
+### Great Places to Eat
```
Comment (githubteacher) [inline places/urbana.md:3]: @brianamarie I probably shouldn't ask you for Italian recommendations??

File: places/urbana.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+### Fun Things to Do
```
Comment (jcronk) [inline places/urbana.md:8]: There are fun things to do in IL? :smiling_imp:

File: places/urbana.md (hunk 1 above)
Comment (brianamarie) [inline places/urbana.md:3]: Italian is fine, let's bring umbrellas and wet wipes!

File: places/urbana.md (hunk 2 above)
Comment (brianamarie) [inline places/urbana.md:8]: > There are fun things to do in IL?

Absolutely, @jcronk. 😜 I'll add some suggestions in my next round of edits.

**Comment by githubteacher** [comment]
@brianamarie this looks great :shipit:

**Pull request merged**